tweets/unconference_mentoring.tweet

```diff
@@ -0,0 +1 @@
+Join us on Thursday June 24th at 13 CEST on Zoom (sign up for the unconference (https://nordic-rse.org/events/2021-online-unconference) for zoom details) for an informal mentoring chat to finalize or even initialize your contribution to #NordicRSE unconference next week #RSEng
```

Comment:
```suggestion
Join us on Thursday June 24th at 13 CEST for an informal mentoring chat to finalize or even initialize your contribution to #NordicRSE unconference next week #RSEng (register for the unconference for connection details): https://nordic-rse.org/events/2021-online-unconference
```

Reply:
Reordered to have the important info first and also to show again the website preview card.